src/components/SlateEditor/plugins/link/LinkForm.jsx

```diff
@@ -95,9 +113,9 @@ class LinkForm extends Component {
               name="href"
               description={`${t('form.content.link.description')} ${
                 config.ndlaFrontendDomain
-              }`}
+              }. ${t('form.content.link.descriptionPartTwo')}`}
```

Comment:
Så lenge teksten berre appendes til den forrige er det bedre å utvide eksisterende tekst enn å legge til en ny. Den brukes uansett kun en plass.

Reply:
Støtte for å sette inn tekst i et felt finnes slik at du kan inserte en parameter inn i en streng.